slides/markdown/team_identity.md

```diff
@@ -30,4 +30,4 @@ Form a straight line across the room in order of relative technical experience i
 
 <!-- .slide: data-background-image="images/chef-background.png", class="white-style" -->
 ### DevOps practices used in this section:
-- Team Identity
```

Comment:
@rmarting can you revert this change? The practice is in fact "Team Identity". "Team Sentiment" is a a different practice. On the other hand, "Team Identity" is not describe in the OPL yet, so I opened an issue in the OPL repo to add it: https://github.com/openpracticelibrary/openpracticelibrary/issues/1440#issuecomment-796767009

Reply:
Removed the OPL practice. Waiting to have the new one to link it.